syntax = "proto3";

package autoxtime.proto;

message Car {
  optional int64 car_id = 1;
  optional int64 car_number = 2;
  optional int64 year = 3;
  optional string make = 4;
  // primary key 1a
  optional string model = 5;
  // primary key 1b
  optional string color = 6;
  optional string sponsor = 7;
  optional string tire_brand = 8;
  // primary key 1c
  optional int64 car_class_id = 9;
  optional int64 driver_id = 10;
  optional int64 organization_id = 11;
}
